evergreen.yml

```diff
@@ -64,6 +64,7 @@ functions:
   "generate expansions":
     - command: shell.exec
       params:
+        silent: true
```

Comment:
Why not adding `set +o xtrace` after `set -o errexit` like you did for the Beta branch?

Reply:
xtrace won't matter when it's silent. I've abandoned the beta branch updates until the sign task is working here. I have silent: false right now for signing while I try to figure out why it's not working

Reply:
We can also use external scripts, but I think silent keeps is simpler.